src/lib/ecomm/types.ts

```diff
@@ -408,7 +408,7 @@ export interface GetPaymentsResponse {
 export interface ConfirmPaymentsPayload {
   site_name: string,
   session_id: string,
-  state: 'PROCESSING' | 'PROCESSED' | 'INVALIDATED' | 'FAILED',
+  state: 'PROCESSING' | 'PROCESSED' | 'INVALIDATED' | 'FAILED' | string,
```

Comment:
Why add this string type. If it can be any string, we shouldn't have the union of specific strings.